test/avocado/selenium-sosreport.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/python
+
+# we need to be able to find and import seleniumlib, so add this directory
+import os
+import sys
+import time
+machine_test_dir = os.path.dirname(os.path.abspath(__file__))
+sys.path.insert(1, machine_test_dir)
+
+from avocado import main
+from avocado.utils import process
+import libdisc
+from seleniumlib import *
+
+class SosReportingTab(SeleniumTest):
+    """
+    :avocado: enable
+    """
+    def test10SosReport(self):
+        self.login()
+        self.wait_id("sidebar")
+        self.wait_id("tools-panel",cond=invisible)
+        self.click(self.wait_link('Tools', cond=clickable))
+        self.wait_id("tools-panel")
+        self.click(self.wait_link('Diagnostic report', cond=clickable))
+        self.wait_frame("sosreport")
+        self.wait_text("This tool will collect system configuration and diagnostic")
+        self.click(self.wait_xpath('//button[@data-target="#sos"]', cond=clickable))
+        self.wait_id("sos")
+        self.wait_text("Generating report")
+        process.run("pgrep sosreport", shell=True)
+        while True:
+            try:
+                process.run("pgrep sosreport", shell=True)
+                self.wait_text("Generating report", overridetry=5)
+            except:
+                break
+            time.sleep(1)
+        element = self.wait_id("sos-download")
+        self.wait_xpath('//button[contains(text(), "%s")]' % "Download", cond=clickable, baseelement=element)
+        self.click(self.wait_id("sos-cancel", cond=clickable))
```

Comment:
Yep, it would be perfect. but it has more troubles,
I tried to find solution here: https://github.com/jscotka/cockpit/tree/custom_profiles_browser
for earch browser I have to create own profile/options what are then send to that browser, and each browser has own way how to paste there these options :-) see commits. I was able to do it working in firefox, for chrome it is bigger issue. 
So another possibility is to make special test only for firefox, and then try it download there.